src/platforms/ios/iosadjusthelper.mm

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "iosadjusthelper.h"
+#include "logger.h"
+#include "constants.h"
+
+#import <AdjustSdk/Adjust.h>
+
+namespace {
+
+Logger logger(LOG_IOS, "IOSAdjustHelper");
+
+}  // namespace
+
+void IOSAdjustHelper::initialize() {
+
+  NSString *adjustToken = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"ADJUST_SDK_TOKEN"];
+
+  if(!adjustToken.length) {
```

Comment:
Does this mean, if adjustToken is empty? We want the opposite, right?
Plus, this is a pointer. What about if we do:

`if (adjustToken) {`

Reply:
I had it mixed up so I removed the `!`. 
Using `adjustToken.length` checks the `adjustToken` for being nil and length 0.